### PR TITLE
Improve warning when creating invoice without wallet

### LINKED ISSUE #2834: Improve the warning when creating an invoice without the configured wallet
This warning is very unclear to a newbie.

Instead of `You need to configure the derivation scheme in order to create an invoice`

We should say something like: `To create an invoice, you need to set up your wallet first`, possibly with the link to the store wallet setup. 

To replicate:
1. Create a store
2. Do not create a wallet
3. Try to create an invoice

Screenshot 2021-09-02 at 11 34 11

### PR DESCRIPTION
**Pull request by bolatovumar**
Moved the error message from the form element up to the top of the page so user doesn't have to scroll down the page to see the error message. Also added a link to the store page where user can set up their wallet.

**Before:**

Screen Shot 2021-09-05 at 7 06 38 PM

**After:**

https://user-images.githubusercontent.com/1934678/132150849-064da593-15e0-423d-87a9-122e2dd95743.mov

close #2834

### CODE REVIEW

**Comment by pavlenex** [comment]
@dstrukt @dennisreimann is there anything we can do to make a link better visible in the red warning box? Yellow on red does seem odd to me.

File: BTCPayServer/Controllers/InvoiceController.UI.cs [hunk 1]
```diff
@@ -843,7 +843,12 @@ public async Task<IActionResult> CreateInvoice(CreateInvoiceModel model, Cancell
 
             if (!store.GetSupportedPaymentMethods(_NetworkProvider).Any())
             {
-                ModelState.AddModelError(nameof(model.StoreId), "You need to configure the derivation scheme in order to create an invoice");
+                TempData.SetStatusMessageModel(new StatusMessageModel()
+                {
+                    Severity = StatusMessageModel.StatusSeverity.Error,
+                    Html = $"To create an invoice, you need to <a href='{Url.Action(nameof(StoresController.UpdateStore), "Stores", new { storeId = store.Id })}' class='link-warning'>set up your wallet</a> first",
```
Comment (dennisreimann) [inline BTCPayServer/Controllers/InvoiceController.UI.cs:849]:
```suggestion
                    Html = $"To create an invoice, you need to <a href='{Url.Action(nameof(StoresController.UpdateStore), "Stores", new { storeId = store.Id })}' class='alert-link'>set up your wallet</a> first",
```

**Review by dennisreimann — CHANGES_REQUESTED** [review]
Fixes the link color

File: BTCPayServer/Controllers/InvoiceController.UI.cs (hunk 1 above)
Comment (bolatovumar) [reply BTCPayServer/Controllers/InvoiceController.UI.cs]: Thanks! Updated here: 1e3994aaf780fc004a1f91c0cef0f7352100cff7


https://user-images.githubusercontent.com/1934678/132249482-d92734cb-8d59-4276-b462-f4dffaaa1409.mov

**Review by dennisreimann — APPROVED** [review]

**Review by pavlenex — APPROVED** [review]
LGMT

**Pull request merged**